apiary.apib

```diff
@@ -1206,17 +1988,15 @@ Fetches an event by identifier
     Event not found
 
     + Body
-    
-## Group Events2
 
+## Group Event
+__ⓘ Note:__ Available only for Hermes node. Lagacu endpoint event2 should be used.
```

Comment:
One more misspell - 'Lagacu' )  Did you use a tool to spellcheck?

Reply:
File was checked with spellchecker now.